samples/nrf9160/aws_iot/README.rst

```diff
@@ -184,3 +183,4 @@ It uses the following `sdk-nrfxlib`_ library:
 In addition, it uses the following sample:
 
 * :ref:`secure_partition_manager`
+* :ref:`Trusted Firmware-M <ug_tfm>`
```

Comment:
This is listing a user guide under samples. May be this should be separate sentence like, "The sample also uses :ref:`Trusted Firmware-M <ug_tfm>`. "? Same comment applies to all the samples edited in the PR.

Reply:
I think it's not quite accurate to call TF-M a sample. Maybe this could read something like "secure firmware components" instead, like in `tfm_spm_thingy91.txt`?

Reply:
@umapraseeda , @jtguggedal : I have changed from "samples" to "secure firmware components". Hope this is fine now.

Reply:
Looks good to me, thanks!

Reply:
@divipillai  May be we should consider adding this line into the sample template also.

Reply:
Created https://projecttools.nordicsemi.no/jira/browse/NCSDK-15486